extensions/batch/chathistory.md

```diff
@@ -0,0 +1,36 @@
+---
+title: IRCv3 CHATHISTORY Batch Type
+layout: spec
+copyrights:
+  -
+    name: "Peter Powell"
+    period: "2015"
+    email: "petpow@saberuk.com"
+---
+## CHATHISTORY Batch Type
+
+This document describes the format of the CHATHISTORY batch type.
+
+The CHATHISTORY batch type takes one parameter. This parameter contains the
+target of messages within the batch. The target MUST either be the nick of the
+receiving client for private messages or the name of a channel which the
```

Comment:
`znc.in/playback` uses nick of the person who the receiving client was communicating with instead, so for 2 different nicks, it will be 2 different batches.